src/panel/servicePanel.jsx

```diff
@@ -0,0 +1,64 @@
+/* global _ */
+import React from 'react';
+import Panel from 'src/components/ui/Panel';
+import CategoryService from '../adapters/category_service';
+import nconf from '@qwant/nconf-getter';
+
+class ServicePanel extends React.Component {
+  render() {
+    return <Panel
+      resizable
+      title={_('Qwant Maps services', 'service panel')}
+      minimizedTitle={_('Show Qwant Maps services', 'service panel')}
+      className={'service_panel'}
```

Comment:
The `{ }` are not necessary. There are ESLint rules specific to JSX to help for this kind of stuff, maybe it's time to introduce them :)